package.json

```diff
@@ -204,7 +204,7 @@
     "balena-errors": "^4.7.1",
     "balena-image-fs": "^7.0.6",
     "balena-image-manager": "^7.0.3",
-    "balena-preload": "^10.4.8",
+    "balena-preload": "^10.4.10",
```

Comment:
If you wanted version 10.4.10 exactly, this should be `10.4.10` instead of `^10.4.10`.